implementer-guide/configuration/security.md

```diff
@@ -3,29 +3,64 @@ title: Security
 page_title: Security
 description: Security Settings
 slug: security
-tags: security,settings
+tags: security,settings,report,server
 published: True
 position: 850
 ---
 
 # Security
 
+>caption The Security tab of the Report Server Configuration page:
+
+![The Report Server Configuration page.](../../images/report-server-images/security-rate-limiter-configuration.png)
+
 ## Rate Limiter
 
+Introduced with [2024 Q3 (10.2.24.806)](https://www.telerik.com/support/whats-new/report-server/release-history/progress-telerik-report-server-2024-q3-10-2-24-806).
+
 Rate limiting is a technique used to control the rate of incoming requests to an API. It sets limits on the number of requests that can be made by an IP address to a specific anonymous endpoint within a defined time period (window). The rate limiter is applied only to non-authenticated users. The Guest special account is not affected by the rate limiter.
 
 When the client has exhausted the number of requests allowed, a 409 "Conflict" response with "The endpoint "{endpointName}" is currently not accessible." message is returned from the server.
 
-![security-rate-limiter-configuration](../../images/report-server-images/security-rate-limiter-configuration.png)
-
 ### Enable rate limiter
 
-Enables/Disables the rate limiter. By default the value is set to enabled.
+Enables/Disables the rate limiter. By default, the value is set to `enabled`.
 
 ### Window
 
-Specifies the time window in milliseconds that takes in the requests. It must be set to a value greater than 0. By default the value is set to 1000 milliseconds.
+Specifies the time window in milliseconds that takes in the requests. It must be set to a value greater than 0. By default, the value is set to `1000 milliseconds`.
 
 ### Permit Limit
 
-The maximum permitted number of requests that can be allowed in a time window. It must be set to a value greater than 0. By default the value is set to 1.
\ No newline at end of file
+The maximum permitted number of requests that can be allowed in a time window. It must be set to a value greater than 0. By default, the value is set to `1`.
+
+## Encryption
+
+Introduced with 2024 Q4 (10.3.24.1112).
+
+With enabled encryption, the Report Server assets are stored in the [Report Server Storage]({%slug storage-settings%}) in encrypted form. The encryption algorithm uses a pair of keys, __Main Key__ and __Backup Key__ created during the initial configuration of the Report Server:
```

Comment:
The sentence sounds like the assets were not encrypted until now, which is not correct. They were encrypted, but with a weaker algorithm. Now we're introducing industry-standard encryption routines.
How about: 
"The sensitive assets in Report Server are always stored in encrypted form. Now the encryption functionality is leveraged to use industry-standard encryption routines. The encryption algorithms use a pair of keys..."